bedrock/mozorg/views.py

```diff
@@ -128,12 +128,14 @@ def namespaces(request, namespace):
 def home_view(request):
     locale = l10n_utils.get_locale(request)
 
-    ctx = {
-        "ftl_files": ["mozorg/home", "mozorg/home-mr2-promo"],
-        "add_active_locales": ["de", "fr"],
-    }
+    variation = request.GET.get("v", None)
+
+    if variation not in ["1", "2"]:
+        variation = None
+
+    ctx = {"ftl_files": ["mozorg/home", "mozorg/home-mr2-promo"], "add_active_locales": ["de", "fr"], "variation": variation}
 
-    if locale.startswith("en-"):
+    if locale.startswith("xs-"):
```

Comment:
Is this to deliberately disable the old en homepages (Contentful or templated)? If so, might as well just drop the entire if clause and start with `if locale == "de":`

Reply:
I made it a typo when I first started working on this PR so the rest-of-world homepage would appear instead of Contentful. It was a janky hacky way of getting things rolling without messing with this Python file too much. I'm planning on cleaning all this up before having it reviewed